### PR TITLE
Model validation

### PR DESCRIPTION
**Pull request by MoritzWeber0**
Adds a Gitlab CI template for model validation

### CODE REVIEW

File: ci-templates/gitlab/exporter.yml [hunk 1]
```diff
@@ -36,34 +36,34 @@ export-to-t4c:
...
-    HTTP_PASSWORD: ${HTTP_PASSWORD}
+    # HTTP_PASSWORD: ${HTTP_PASSWORD}
 
     # HTTP Port (port for REST API endpoint)
     # Defaults to 8080
-    HTTP_PORT: 8080
+    # HTTP_PORT: 8080
 
     # Log level. All Python `logging` levels are allowed.
     # Defaults to INFO
```
Comment (jamilraichouni) [inline ci-templates/gitlab/exporter.yml:69]: This does not belong to the topic "Model Validation". Nevertheless, it is quicker to "reinschummel" it here.

File: ci-templates/gitlab/model-validation.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+model-validation:
+  image:
+    name: $DOCKER_REGISTRY/capella/cli:${CAPELLA_DOCKER_IMAGES_TAG}
+    entrypoint: [""]
+  script:
+    - git fetch
+    - if [ ! -z $CI_COMMIT_BRANCH ]; then git reset --hard origin/$CI_COMMIT_BRANCH; fi
+    - BUILD_DIR=$(pwd)
+    - if [ ! -f ".project" ]; then echo "The model validator requires a '.project' file!" && echo 1; fi
```
Comment (jamilraichouni) [inline ci-templates/gitlab/model-validation.yml:12]:
```suggestion
    - if [ ! -f ".project" ]; then echo "The model validator requires a '.project' file!" && exit 1; fi
```

**Review by jamilraichouni — CHANGES_REQUESTED** [review]

**Review by dominik003 — APPROVED** [review]

File: ci-templates/gitlab/exporter.yml (hunk 1 above)
Comment (MoritzWeber0) [reply ci-templates/gitlab/exporter.yml]: Was not intended. I've changed the base branch, it's not part of the PR anymore.

File: ci-templates/gitlab/model-validation.yml (hunk 2 above)
Comment (MoritzWeber0) [reply ci-templates/gitlab/model-validation.yml]: Fixed.

**Review by jamilraichouni — APPROVED** [review]
SGTM!

**Pull request merged**